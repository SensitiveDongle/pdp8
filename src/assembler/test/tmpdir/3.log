Undefined symbol PUTC used at line 6
*** 1 errors in assembly
