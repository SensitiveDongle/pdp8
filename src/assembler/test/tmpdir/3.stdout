  0x000: 0xE80    1: 		CLA
  0x001: 0x200    2: 		TAD   C
  0x002: 0xA00    3: 		JMP   NX
  0x003: 0x035    4: C,		'5'
  0x004: 0x000    5: D,		0
  0x005: 0x800    6: NX,		JMS  PUTC
